src/l500/l500-private.cpp

```diff
@@ -100,5 +101,49 @@ namespace librealsense
             :_hw_monitor(hw_monitor), _option(opt)
         {
         }
+
+        flash_structure get_rw_flash_structure(const uint32_t flash_version)
+        {
+            switch (flash_version)
+            {
+            case 103: return { 1, { 40, 320, 321, 326, 327, 54} };
```

Comment:
how it is going to be maintained - pls refactor

Reply:
comments added